sentinel-adapter/sentinel-motan-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/motan/MotanAppContextFilter.java

```diff
@@ -0,0 +1,31 @@
+package com.alibaba.csp.sentinel.adapter.motan;
+
+import com.weibo.api.motan.core.extension.Activation;
+import com.weibo.api.motan.core.extension.SpiMeta;
+import com.weibo.api.motan.filter.Filter;
+import com.weibo.api.motan.rpc.Caller;
+import com.weibo.api.motan.rpc.Request;
+import com.weibo.api.motan.rpc.Response;
+import com.weibo.api.motan.rpc.RpcContext;
+
+import java.util.Map;
+
+/**
+ * program: sentinel-parent
+ * description: ${description}
+ * author: zxn
+ * create: 2020-10-28 00:28
+ **/
+@SpiMeta(name = "motanAppContext")
+@Activation(sequence = 2)
+public class MotanAppContextFilter implements Filter {
```

Comment:
This filter doesn't seem to work, is the `key = MotanConstants.NODE_TYPE_REFERER` missing? 
like `SentinelMotanConsumerFilter`.

Reply:
yes， I forget to add it